app/src/features/city/subsectorSlice.ts

```diff
@@ -2,111 +2,34 @@ import { RootState } from "@/lib/store";
 import { SubSectorAttributes } from "@/models/SubSector";
 import { PayloadAction, createSlice } from "@reduxjs/toolkit";
 import { SubCategory, SubCategoryAttributes } from "@/models/SubCategory";
-import { SectorFormSchema } from "@/util/form-schema";
+import { manualInputHierarchySchema } from "@/util/form-schema";
 
 export type Methodology = {
-  methodologyId: string;
-  methodologyName: string;
-  description: string;
-  inputRequired: string[];
-  disabled: boolean;
-};
+  id:string,
+  activities: any;
+  suggestedActivities: any
+  methodologyName?: string,
+  description?: string,
+  inputRequired?: string[],
+  disabled:boolean,
+}
 
 export type ActivityDataScope = {
-  scope: number;
-  formInputs: any;
-  methodologies: Methodology[];
-};
+  direct_measure: {
```

Comment:
Direct measure is just another activity methodology for our data model, and we save the 3 inputs as keys `co2_value`, `ch4_value`, `n2o_value` in the `activitityData` column of ActivityValue.